.pylintrc

```diff
@@ -56,7 +56,7 @@ confidence=
 # --enable=similarities". If you want to run only the classes checker, but have
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
-disable=suppressed-message,arguments-differ,wildcard-import,locally-disabled,duplicate-code,no-else-return
+disable=suppressed-message,arguments-differ,wildcard-import,locally-disabled,duplicate-code,no-else-return,no-member
```

Comment:
In the operator class files, I used self._dict_.update(kwargs) to update class variables with kwargs. Because of this, now pylint throws a no-member error on commit